.circleci/config.yml

```diff
@@ -201,14 +229,15 @@ workflows:
             - vblocks-js
           name: Build Checks
       - run-integration-tests:
-          name: Integration Tests <<matrix.browser>> <<matrix.bver>>
+          name: Integration tests <<matrix.browser>> <<matrix.bver>>
           context:
             - dockerhub-pulls
             - vblocks-js
           matrix:
             parameters:
               browser: ["chrome", "firefox"]
               bver: ["beta", "unstable", "stable"]
+              build_label: ["Integration tests"]
```

Comment:
```suggestion
              build_label: ["Pull request"]
```

Reply:
Not sure if this will look good. check the example output here https://twilio.slack.com/archives/C042FCK9WSD/p1688595179532909